src/main/java/it/eng/idsa/businesslogic/service/impl/ClearingHouseServiceImpl.java

```diff
@@ -143,7 +143,7 @@ public boolean registerTransaction(Message messageForLogging, String contractAgr
 			logger.info("Sending Data to the Clearing House {} ...", endpoint);
 			response = sendDataToBusinessLogicService.sendMessageFormData(endpoint, multipartMessage, getBasicAuth());
 			
-			logger.info("Data [LogMessage.id={}] sent to the Clearing House {}", multipartMessage.getHeaderContent().getId(), endpoint);
+			logger.info("Data sent to the Clearing House");
```

Comment:
This is not considered as sensitive data, we could leave that in log

Reply:
Agreed, I will revert it